### PR TITLE
Db tickets

### LINKED ISSUE #48842: DB ticket senders and recipients aren't being saved
I blew it.

### PR DESCRIPTION
**Pull request by nfreader**
Hotfix, Fixes #48842

### CODE REVIEW

File: code/controllers/subsystem/blackbox.dm [hunk 1]
```diff
@@ -1,353 +1,355 @@
...
-	y_coord = sanitizeSQL(y_coord)
-	z_coord = sanitizeSQL(z_coord)
-	last_words = sanitizeSQL(last_words)
-	suicide = sanitizeSQL(suicide)
-	map = sanitizeSQL(map)
-	var/datum/DBQuery/query_report_death = SSdbcore.NewQuery("INSERT INTO [format_table_name("death")] (pod, x_coord, y_coord, z_coord, mapname, server_ip, server_port, round_id, tod, job, special, name, byondkey, laname, lakey, bruteloss, fireloss, brainloss, oxyloss, toxloss, cloneloss, staminaloss, last_words, suicide) VALUES ('[sqlpod]', '[x_coord]', '[y_coord]', '[z_coord]', '[map]', INET_ATON(IF('[world.internet_address]' LIKE '', '0', '[world.internet_address]')), '[world.port]', [GLOB.round_id], '[SQLtime()]', '[sqljob]', '[sqlspecial]', '[sqlname]', '[sqlkey]', '[laname]', '[lakey]', [sqlbrute], [sqlfire], [sqlbrain], [sqloxy], [sqltox], [sqlclone], [sqlstamina], '[last_words]', [suicide])")
-	if(query_report_death)
-		query_report_death.Execute(async = TRUE)
-		qdel(query_report_death)
+SUBSYSTEM_DEF(blackbox)
```
Comment (Cyberboss) [inline code/controllers/subsystem/blackbox.dm]: NEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEDDDDD!!!!!

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/blackbox.dm (hunk 1 above)
Comment (nfreader) [reply code/controllers/subsystem/blackbox.dm]: Oh good god holy shit

**Comment by Tharcoonvagh** [comment]
Boy oh boy

**Comment by Cyberboss** [comment]
what did he mean by this?

**Pull request closed without merging**